exporters/trace/jaeger/jaeger.go

```diff
@@ -47,6 +46,8 @@ type options struct {
 	// RegisterGlobal is set to true if the trace provider of the new pipeline should be
 	// registered as Global Trace Provider
 	RegisterGlobal bool
+
+	Disabled bool
```

Comment:
It's unclear why we should support a disabled Exporter, as opposed to use of a Noop Meter provider.  NewExportPipeline could be adjusted to check the disabled flag and return a Noop provider.  `NewRawExporter` can returned an exporter that ignores the DISABLED setting.

Reply:
I agree that returning a Noop provider is a good idea, but `NewRawExporter` and `NewExportPipeline` use the same `Option`, so the user (developer) may pass the DISABLED setting and find out that is useless.

It's that OK if we just write a comment that `NewRawExporter` will ignore the DISABLED setting?

Reply:
> It's that OK if we just write a comment that NewRawExporter will ignore the DISABLED setting?

I think so, yes.

Reply:
See also #800.